v1/employees/get.md

```diff
@@ -0,0 +1,209 @@
+# Employee List
```

Comment:
Any difference between "list" and "index"?
I've always used index, but list makes sense to me too.

Reply:
Nope, I'm only using it in an attempt to be more descriptive for the end user.